### DISCUSSION TITLE
Refresh widget render on note attribute change?

### DISCUSSION THREAD

**Discussion by nicollasricas** [TriliumNext/Trilium#2918] [Q&A] [answered]
Hey, @zadam,

Is there a way to refresh widget render on note attribute change?

Thanks!

**Comment by zadam** [comment, marked as answer]
Hi, yes, this can be done like this:

https://github.com/zadam/trilium/blob/master/src/public/app/widgets/editability_select.js#L87-L91

Every widget receives an event on every update, widget then decides whether it needs to refresh or not.
